src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs

```diff
@@ -49,21 +48,20 @@ public void SetUp()
             routingTopology = new ConventionalRoutingTopology(true);
             receivedMessages = new BlockingCollection<IncomingMessage>();
 
-            var settings = new SettingsHolder();
-            settings.Set("NServiceBus.Routing.EndpointName", "endpoint");
+            var endpointName = "NServiceBus.Routing.EndpointName";
```

Comment:
Oops, I used the wrong name! Will fix.

I'll make the `ReceiverQueue` change in a separate commit.